docs/software/uv.md

```diff
@@ -82,28 +70,39 @@ To activate the environment, use the command:
 ```
 $ source $UV_ENVS/mycustomenv/bin/activate
 ```
```

Comment:
Now that we have removed `mycustomenv` below. Maybe it is worth spelling it out here for users e.g.

You will know that your environment has been properly loaded when the environment name appears in parenthesis on the command line e.g. `(mycustomenv)`. 

or something like that